app/views/partials/personal_key/_key.html.erb

```diff
@@ -15,4 +12,33 @@
           date: content_tag(:strong, I18n.l(Time.zone.today, format: '%B %d, %Y')),
         ) %>
   </p>
+
+  <br />
+
+  <%= render ClipboardButtonComponent.new(
+        clipboard_text: code,
+        unstyled: true,
+      ) %>
+
+  <%= render ButtonComponent.new(
+        action: ->(**tag_options, &block) do
+          link_to(
+            "data:text/plain;charset=utf-8,#{CGI.escape(code)}",
+            download: 'personal_key.txt',
+            **tag_options,
+            &block
+          )
+        end,
+        icon: :file_download,
+        unstyled: true,
+        class: 'margin-right-2 margin-bottom-2 tablet:margin-bottom-0',
```

Comment:
From the design, it seems like there should be equal right and left margin here. Do we need the vertical margin? Also, I'm seeing some layout issues on mobile. I'm not entirely sure why, but it seems to clear up by changing the display from inline to inline-block.

```suggestion
        class: 'margin-x-2 display-inline-block',
```

Before|After
---|---
![image](https://user-images.githubusercontent.com/1779930/195846870-69284514-fa6d-4498-9ca9-93fac63cee4b.png)|![image](https://user-images.githubusercontent.com/1779930/195847391-ad4832f4-f008-4fa4-bd36-21dac07455f6.png)